public/app/features/dashboard/containers/DashboardPage.tsx

```diff
@@ -97,6 +111,69 @@ export class UnthemedDashboardPage extends PureComponent<Props, State> {
 
   private forceRouteReloadCounter = 0;
   state: State = this.getCleanState();
+  maxFileSize = 500000;
```

Comment:
Would be good to pull this into a module level constant (i.e. MAX_FILESIZE or similar)